README.md

```diff
@@ -11,3 +11,41 @@ Health Check
 ---
 
 To see your applications health enter url `http://localhost:8081/healthcheck`
+
+
+ENV Variables
+--- 
+
+To use DropwizardTheITCrowd API you will need to have the following environment variables
+
+- DB_USERNAME &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;# Database Account Username
+- DB_PASSWORD &nbsp;&nbsp;&nbsp;&nbsp;&nbsp; # Database Account Password
+- DB_HOST &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; #  Database Hostname 
+- DB_NAME &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; # Database Schema Name
+
+These have been used to connect to the aws database.
+
+        user = System.getenv("DB_USERNAME");
```

Comment:
It might be better to set this up as `export` commands

Reply:
Fixed